package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "git-pr-release-action",
-  "version": "1.0.0",
+  "name": "@splashthat/git-pr-release-action",
+  "version": "1.0.2",
```

Comment:
Why?

Reply:
I guess you miss `1.0.1`. The reason why it doesn't appear in this PR is that I made a version `1.0.1` in the branch, but later I bumped the version again to `1.0.2`.
If you see the repo, you will still be able to find the tag `v1.0.1`.